sale_quotation_number/__manifest__.py

```diff
@@ -24,7 +24,7 @@
 
 {
     'name': 'Sale Quotation Numeration',
-    'version': '8.0.1.1.0',
+    'version': '10.0.1.1.0',
     'category': 'Sales Management',
     'summary': "Different sequence for sale quotations",
     'description': """
```

Comment:
Please move module description to README file, following the template
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
and replacing 'openerp' with 'odoo'